2012/04/25 21:47:40 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/arden/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://arden/index.php?r=gii/controller
---
2012/04/25 21:52:52 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/arden/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://arden/index.php?r=gii/controller
---
2012/04/25 21:52:53 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/arden/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://arden/index.php?r=gii/controller
---
2012/04/25 21:53:10 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO)' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/arden/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://arden/index.php?r=gii/controller
---
2012/04/25 21:53:11 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO)' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/arden/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://arden/index.php?r=gii/controller
---
2012/04/25 21:53:11 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO)' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/arden/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://arden/index.php?r=gii/controller
---
2012/04/25 21:53:12 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO)' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/arden/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://arden/index.php?r=gii/controller
---
2012/04/25 21:53:13 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO)' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/arden/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://arden/index.php?r=gii/controller
---
2012/04/25 21:53:13 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO)' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/arden/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://arden/index.php?r=gii/controller
---
2012/04/25 21:53:13 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO)' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/arden/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://arden/index.php?r=gii/controller
---
2012/04/25 21:53:14 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO)' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/arden/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://arden/index.php?r=gii/controller
---
2012/04/25 21:53:28 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'arden'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/arden/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://arden/index.php?r=gii/controller
---
2012/04/25 21:53:29 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'arden'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/arden/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://arden/index.php?r=gii/controller
---
2012/04/25 21:53:29 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'arden'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/arden/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://arden/index.php?r=gii/controller
---
2012/04/25 21:53:29 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'arden'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/arden/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://arden/index.php?r=gii/controller
---
2012/04/25 21:53:29 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'arden'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/arden/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://arden/index.php?r=gii/controller
---
2012/04/25 21:53:30 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'arden'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/arden/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://arden/index.php?r=gii/controller
---
2012/04/25 21:53:30 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'arden'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/arden/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://arden/index.php?r=gii/controller
---
2012/04/25 22:11:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to find the code you requested.' in /var/www/yii/framework/gii/CCodeGenerator.php:90
Stack trace:
#0 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionCode()
#1 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#2 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#3 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#4 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('code')
#5 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model/code')
#6 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#7 /var/www/arden/index.php(13): CApplication->run()
#8 {main}
REQUEST_URI=/index.php?r=gii/model/code&id=0
HTTP_REFERER=http://arden/index.php?r=gii/model
---
2012/04/26 23:40:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "service".' in /var/www/yii/framework/web/CController.php:484
Stack trace:
#0 /var/www/yii/framework/web/CController.php(271): CController->missingAction('service')
#1 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('service')
#2 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('site/service')
#3 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /var/www/arden/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/service
HTTP_REFERER=http://arden/
---
2012/04/26 23:40:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "service".' in /var/www/yii/framework/web/CController.php:484
Stack trace:
#0 /var/www/yii/framework/web/CController.php(271): CController->missingAction('service')
#1 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('service')
#2 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('site/service')
#3 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /var/www/arden/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/service
HTTP_REFERER=http://arden/
---
2012/04/26 23:40:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "service".' in /var/www/yii/framework/web/CController.php:484
Stack trace:
#0 /var/www/yii/framework/web/CController.php(271): CController->missingAction('service')
#1 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('service')
#2 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('site/service')
#3 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /var/www/arden/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/service
HTTP_REFERER=http://arden/
---
2012/04/26 23:40:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "service".' in /var/www/yii/framework/web/CController.php:484
Stack trace:
#0 /var/www/yii/framework/web/CController.php(271): CController->missingAction('service')
#1 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('service')
#2 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('site/service')
#3 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /var/www/arden/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/service
HTTP_REFERER=http://arden/
---
